src/app/pages/details/details.scss

```diff
@@ -0,0 +1,228 @@
+@import 'variables';
+
+$light-gray: lighten(color(neutral-light), 36%);
+$get-this-book-width: 25rem;
+$secondary-button: lighten(color(neutral-dark), 12%);
+$text-black: rgba(color(neutral-darkest), 0.87);
+$text-secondary: rgba(color(neutral-darkest), 0.7);
+
+.text-content {
+    color: $text-black;
+    font-family: Lato;
+    font-size: 1.7rem;
+    max-width: (120rem + #{$get-this-book-width});
+    position: relative;
+
+    > .block {
+        padding: 5rem (2rem + #{$get-this-book-width}) 5rem 10rem;
+
+        &.book-info {
+            background-image: $page-top-gradient;
+        }
+
+        &.endorsement {
+            background-color: color(green);
+        }
+
+        .links > a {
+            display: block;
+            margin-top: 1rem;
+        }
+    }
+}
+
+a,
+.tab > label,
+.option {
```

Comment:
Eek, I missed this, but it's affecting EVERY anchor tag on the entire page!  This all needs to be moved inside a .details block.